### ISSUE TITLE
recognize model

### ISSUE THREAD

**Issue by Jerry-Kon** [InftyAI/llmlite#30]
User can custom the name of model path. Only using path name can not recognize which model to choose.
We can recognize model , by string "_name_or_path" in config.json which is included in model files.

**Comment by Jerry-Kon** [comment]
![image](https://github.com/InftyAI/llmlite/assets/92307128/0ee8b1b5-a6ea-4c87-8108-03a608996dd2)

**Comment by kerthcet** [comment]
Yes, we should support `config.json` in the future, it's recommended by the hf community.
